src/server/bootstrap.rs

```diff
@@ -174,6 +174,21 @@ pub fn print_access_record(gsettings: &GlobalSettings, server_dir: &Path, record
     assert!(print_stdout(table).is_ok());
 }
 
+pub async fn info_hq_server(gsettings: &GlobalSettings) -> anyhow::Result<()> {
+    match get_server_status(gsettings.server_directory()).await {
+        Err(_) | Ok(ServerStatus::Offline(_)) => anyhow::bail!("No online server found"),
+        Ok(ServerStatus::Online(_)) => {
```

Comment:
The `Online` state already gives you the opened access record, there's no need to load it again.